### PR TITLE
Rescale Google Developers | Go-jek logo

### LINKED ISSUE #188: Facing errors in starting the server
Followed the instructions in the readme.md to run the blog site on the server, faced this error repeatedly with every blog post that we have. I think the instructions are bit outdated.

```
ERROR: Could not process blog/announce-oyo-rooms-as-accomodation-partner-2018.md
  | Failed loading extension "codehilite(css_class=highlight)".
  |___
  | Traceback (most recent call last):
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/pelican/generators.py", line 502, in generate_context
  |     context_sender=self)
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/pelican/readers.py", line 459, in read_file
  |     content, reader_metadata = reader.read(path)
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/pelican/readers.py", line 236, in read
  |     self._md = Markdown(extensions=self.extensions)
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/markdown/core.py", line 100, in __init__
  |     configs=kwargs.get('extension_configs', {}))
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/markdown/core.py", line 126, in registerExtensions
  |     ext = self.build_extension(ext, configs.get(ext, {}))
  |   File "/home/vipulgupta2048/.virtualenvs/pycon-blog/local/lib/python2.7/site-packages/markdown/core.py", line 166, in build_extension
  |     module = importlib.import_module(ext_name)
  |   File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
  |     __import__(name)
  | ImportError: Failed loading extension "codehilite(css_class=highlight)".
```

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (vipulgupta2048) <vipulgupta2048@gmail.com>

- Fix Google Logo - rescaled to fit Desktops
- Deleted extra lines
- Fix #188 - Updated pelican version. (Thanks @ananyo2012) 

:hatching_chick:

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@ananyo2012  Added Google Devs link

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-pelican==3.5.0
+pelican==3.7.1
```
Comment (realslimshanky) [inline requirements.txt]: I guess this change wasn't required. Can you please remove this from the pull request?

Comment (vipulgupta2048) [reply requirements.txt]: Refer #188

Comment (realslimshanky) [reply requirements.txt]: @vipulgupta2048 let's work this out in a separate PR as it would be easier to test things with the new dependency.

**Comment by realslimshanky** [comment]
@vipulgupta2048 :taco:

**Pull request merged**